File: JavaScript/M03_JavaScriptProgrammingBasics/L03_ConditionalStatementsAdvanced/Lab/P01_DayOfWeek.js
function dayOfTheWeek(input) {
    let num = parseInt(input[0]);

    let day = {
        1: "Monday",
        2: "Tuesday",
        3: "Wednesday",
        4: "Thursday",
        5: "Friday",
        6: "Saturday",
        7: "Sunday"
    }

    if (day[num]) {
        let theDay = day[num];
        console.log(theDay);
    } else {
        console.log("Error");
    }
}

File: C++/M01_ProgrammingBasics/L03_ConditionalStatementsAdvanced/Lab/Solutions/P01_DayOfWeek.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {

	int n;
	cin >> n;

	string output;
	switch (n) {
	case 1:
		output = "Monday";
		break;
	case 2:
		output = "Tuesday";
		break;
	case 3: 
		output = "Wednesday";
		break;
	case 4:
		output = "Thursday";
		break;
	case 5:
		output = "Friday";
		break;
	case 6:
		output = "Saturday";
		break;
	case 7:
		output = "Sunday";
		break;
	default:
		output = "Error";
		break;
	}

	cout << output << endl;

	return 0;
}
